polus-file-renaming-plugin/plugin.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "File Renaming",
+    "version": "0.1.13",
+    "title": "File Renaming",
+    "description": "Rename and store image collection files in a new image collection",
+    "author": "Melanie Parham (melanie.parham@axleinfo.com)",
```

Comment:
Thanks for the clarification. Because I am still in the middle of being transferred to NCATS from NIAID, I do not have access to my previous NIH email. However, I can add it later after I regain access.